docs/Chipyard-Basics/Initial-Repo-Setup.rst

```diff
@@ -66,6 +66,22 @@ You can put this in your ``.bashrc`` or equivalent environment setup file to get
 
 These variables need to be set for the ``make`` system to work properly.
 
+Pre-built Docker Image
+-------------------------------------------
+
+An alternative to setting up the Chipyard repository locally is to pull the pre-built Docker image from Docker Hub. The image comes with all necessary dependencies and toolchains installed.
```

Comment:
I would mention that this docker image has all toolchains included, all dependencies installed, and Chipyard cloned.

Reply:
I might also mention that the tag of the Docker image corresponds to the Chipyard version that is cloned/setup there.